src/python/gudhi/bottleneck.cc

```diff
@@ -29,7 +29,8 @@ PYBIND11_MODULE(bottleneck, m) {
           py::arg("diagram_1"), py::arg("diagram_2"),
           py::arg("e") = (std::numeric_limits<double>::min)(),
           R"pbdoc(
-    This function returns the point corresponding to a given vertex.
+    Compute the Bottleneck distance between two diagrams.
+    Points at infinity and on the diagonal are supported.
```

Comment:
Uh ? Typo ?
```suggestion
    Points at infinity and on the diagonal are not supported.
```

Reply:
This is the interface for François' version, not Hera. Since there is a precision for hera that it handles infinity but not the diagonal, I thought it would be clearer to also precise things for this version, which I believe handles both.

Reply:
Oops, sorry, I forgot bottleneck was under pybind11.